### PR TITLE
Fix/anim broken

### PR DESCRIPTION
**Pull request by aurelenc**
Add new animation system

All frames of an animation must be specified when adding the "Animated" component (x, y, width, height, delay) via variadic parameters

### CODE REVIEW

File: src/ecs/components/client/Animated.hpp [hunk 1]
```diff
@@ -7,25 +7,69 @@
 
 #pragma once
 
+#include <vector>
+
 namespace ecs::component
 {
-    /**
```
Comment (GuyomT) [inline src/ecs/components/client/Animated.hpp:12]: why u delete the doc !!?

**Review by GuyomT — APPROVED** [review]
doc could be better, LGTM anyway

File: src/ecs/components/client/Animated.hpp (hunk 1 above)
Comment (aurelenc) [reply src/ecs/components/client/Animated.hpp:12]: outdated doc

Comment (aurelenc) [reply src/ecs/components/client/Animated.hpp:12]: updated doc

**Review by Cavonstavant — APPROVED** [review]

**Pull request merged**